gpg/decrypt/decrypt.go

```diff
@@ -58,9 +58,16 @@ func (s *Service) Modify(value []byte) ([]byte, error) {
 		return nil, microerror.Mask(err)
 	}
 
-	promptFunc := func(keys []openpgp.Key, symmetric bool) ([]byte, error) {
-		return []byte(s.pass), nil
-	}
+	promptFunc := func() func([]openpgp.Key, bool) ([]byte, error) {
+		retried := false
+		return func(keys []openpgp.Key, symmetric bool) ([]byte, error) {
+			if !retried {
+				retried = true
+				return []byte(s.pass), nil
+			}
+			return nil, microerror.New("Something went wrong. Probably wrong GPG password")
```

Comment:
I would like to have an error being defined separately as we do with all other errors. Please make sure to align with our conventions there. See also https://github.com/giantswarm/giantswarm/wiki/go-style-guide#errors.